app/controllers/api/databases.php

```diff
@@ -1352,6 +1352,9 @@ function updateAttribute(
         if ($encrypt && !empty($plan) && !($plan['databasesAllowEncrypt'] ?? false)) {
             throw new Exception(Exception::GENERAL_BAD_REQUEST, 'Encrypted string attributes are not available on your plan. Please upgrade to create encrypted string attributes.');
         }
+        if ($encrypt && $size < 150) {
```

Comment:
Let's add a new const in constants.php for the 150, `APP_DATABASE_ENCRYPT_SIZE_MIN`